integration/image/import_test.go

```diff
@@ -19,7 +18,8 @@ func TestImportExtremelyLargeImageWorks(t *testing.T) {
 	skip.If(t, runtime.GOARCH == "arm64", "effective test will be time out")
 	skip.If(t, testEnv.OSType == "windows", "TODO enable on windows")
 
-	client := request.NewAPIClient(t)
+	defer setupTest(t)()
```

Comment:
Wondering if we should spin up a new daemon for this one; I'll check how long this test takes to run

Reply:
```
13:32:12 === RUN   TestImportExtremelyLargeImageWorks
13:34:14 --- PASS: TestImportExtremelyLargeImageWorks (122.92s)
```

Looks like that may be worth considering 👍

Reply:
Opened https://github.com/moby/moby/pull/38480 for running it in parallel